app/build.gradle

```diff
@@ -82,7 +83,7 @@ dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.legacy:legacy-preference-v14:1.0.0'
     implementation 'androidx.gridlayout:gridlayout:1.0.0'
-    implementation 'org.bouncycastle:bcprov-jdk15to18:1.69'
+    implementation 'org.bouncycastle:bcprov-jdk15on:1.69'
```

Comment:
Can't use this. From `jdk15on-1.66` Java 16 (major version 59) is required. So we'll have to keep using `jdk15to18`